libs/core/src/lib/toolbar/toolbar-spacer.component.ts

```diff
@@ -15,6 +15,10 @@ export class ToolbarSpacerComponent {
     @Input()
     width = '0px';
 
+    @HostBinding('style.maxWidth')
+    @Input()
+    maxWidth: string;
```

Comment:
Do we really need this input?
User can apply the `max-width` directly to the host element

Reply:
@dimamarksman if we have an input for width, I don't see a problem with having an input for max-width

Reply:
I'm also do not understand purpose of this.

If user wants fixed spacer - he can set its width in `px`, if he wants responsive one - should use `%`.

Instead of `max-width` i would set `flex-shrink: 1` for spacer to handle `width: 9999px` or any other wrong values.